httpapi/api.go

```diff
@@ -0,0 +1,19 @@
+package httpapi
+
+import (
+	"log"
+
+	"github.com/gin-gonic/gin"
+	"github.com/nibble-4bits/ondemand-go-bootcamp/usecase"
+)
```

Comment:
Normally, when imported, go packages are divided into three main blocks:
1. Standard library packages
1. Internal packages
1. 3rd party packages

You can choose any order, but this is what in my experience I've been using on projects.

```suggestion
import (
	"log"
	
	"github.com/nibble-4bits/ondemand-go-bootcamp/usecase"

	"github.com/gin-gonic/gin"
)
```